app/templates/community/invasion-of-ukraine.hbs

```diff
@@ -0,0 +1,129 @@
+        </li>
+        <li class="text-hero-base mb-2">
+          The Ukrainian people ask the world to respond as harshly as possible to what is happening. Ukraine calls on
+          international partners to immediately apply a new round of meaningful sanctions against the aggressor.
+        </li>
+      </ul>
+
+      <p class="text-hero-base mb-2">
+       Putin called it a "military operation." It is not. It is a full-blown war.
+       Ukraine needs the help of foreign governments in the form of intrusion of the military contingent.
+      </p>
+
+      <div class="inner">
+        <h2>
+          HOW TO SUPPORT UKRAINE
+        </h2>
+
+        <p>
+          Follow and share information about the real situation in Ukraine among your friends (<a
+          href="https://kyivindependent.com">Kyiv Independent</a>; <a href="https://www.instagram.com/svidomi_eng">Svidomi
+          (en)</a>).
+        </p>
+
+        <p>
+          Join the rallies against the Russian Federation in your city!
+          Here you can see the schedule of the rallies <a href="https://www.stopputin.net">Stop Putin</a>
+        </p>
+
+        <h3>
+          Support Ukrainian Army
+        </h3>
+        <ul>
+          <li class="text-hero-base mb-2">
+            <p>
+              🇺🇦 The National Bank of Ukraine has opened a <a
+              href="https://bank.gov.ua/en/news/all/natsionalniy-bank-vidkriv-spetsrahunok-dlya-zboru-koshtiv-na-potrebi-armiyi">special
+              account to raise funds for the Ukrainian Army</a>.
+              Anyone from any country can donate!
+            </p>
+            <p>
+              The account is multi-currency. It can receive fund transfers from international partners, donors, and Ukrainian businesses and citizens.
+            </p>
+          </li>
+          <li class="text-hero-base mb-2">
+            🇺🇦 <a href="https://savelife.in.ua/">SaveLife in UA (Повертайся живим)</a>
+            Kyiv-based charity fund that helps the Ukrainian Army since 2014
+          </li>
+        </ul>
+
+        <h3>
+          Charity and Aid to Ukrainians
+        </h3>
+        <ul>
+          <li class="text-hero-base mb-2">
+            ⛑ <a href="https://www.facebook.com/hospitallers/">Medical fund Hospitallers (Госпітальєри) </a>
+            Volunteering medical organisation
+            AID & Medicaments
+            Technic
+          </li>
+          <li class="text-hero-base mb-2">
+            ⛑️ <a href="https://redcross.org.ua/en/donate/">Red Cross in Ukraine</a>
+            Volunteering medical organisation
+            <a href="https://redcross.org.ua/en/donate/">Donation via website</a>
+            <a
+              href="https://donate.redcrossredcrescent.org/ua/donate/~my-donation?_cv=1&ns_ira_cr_arg=IyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyPdfo1VIVJ%2FpZD9AR%2FKX%2F3rHSedNyCLqqggLX9JI3YCxxHcdrK8UkzvUeEB7%2BXvyzotVDFUMj5hliUTfXP8rDSg8d6%2Bu3HLBlW6%2BsOASk4xntWRMY8aRFODAo3qiRdkNZY%3D">Donation
```

Comment:
Does this need all these query params? What happens if we make this more concise?